packages/marketplace/src/services/web-component.ts

```diff
@@ -0,0 +1,81 @@
+import { fetcher } from '@reapit/elements'
+import { generateHeader } from './utils'
+import { logger } from 'logger'
+
+export interface FetchWebComponentConfigParams {
+  customerId: string
+}
+
+export interface PutWebComponentConfigParams {
+  appointmentLength: number
+  appointmentTimeGap: number
+  appointmentTypes: any
+  customerId: string
+  daysOfWeek: string[]
+  negotiatorIds: string[]
+}
+
+export type WebComponentConfigResult = {
+  appointmentLength: number
+  appointmentTimeGap: number
+  appointmentTypes: any
+  customerId: string
+  daysOfWeek: string[]
+  negotiatorIds: string[]
+} | null
+
+const API = '/dev/v1/web-components-config'
```

Comment:
this one can move to `services/constants.ts`
![image](https://user-images.githubusercontent.com/22660833/82774412-f4456080-9e6e-11ea-98f7-83e124b1c1e6.png)